lib/api/2.0/notification.js

```diff
@@ -39,5 +47,8 @@ var notificationProgressGet = controller(function(req, res){
 
 module.exports = {
     notificationPost: notificationPost,
-    notificationProgressGet: notificationProgressGet
+    notificationProgressPost: notificationProgressPost,
+    //NOTE: in some environment, the HTTP client doesn't support POST, so we have to define
```

Comment:
'environments'  could you help provide the example 'ipxe' explicitly for understanding